OOM in `:nessie-quarkus:test` in CI

### Issue description

```
> Task :nessie-quarkus:test

TestVendedS3CredentialsExpiry > testExpiredVendedCredentials() FAILED
    java.lang.RuntimeException at QuarkusTestExtension.java:642
        Caused by: java.lang.OutOfMemoryError at ZipPath.java:104
```

Example: https://github.com/projectnessie/nessie/actions/runs/9504877624/job/26198502849

--- a/build-logic/src/main/kotlin/nessie-testing.gradle.kts
+++ b/build-logic/src/main/kotlin/nessie-testing.gradle.kts
@@ -107,8 +107,8 @@ tasks.withType<Test>().configureEach {
       }
     )
 
-    minHeapSize = if (testHeapSize != null) testHeapSize as String else "512m"
-    maxHeapSize = if (testHeapSize != null) testHeapSize as String else "1536m"
+    minHeapSize = if (testHeapSize != null) testHeapSize as String else "768m"
+    maxHeapSize = if (testHeapSize != null) testHeapSize as String else "2048m"
   } else if (testHeapSize != null) {
     minHeapSize = testHeapSize!!
     maxHeapSize = testHeapSize!!

